### PR TITLE
Add 'anchor points' in landscape

### LINKED ISSUE #3966: Add "anchor points" in landscape
The idea is to add "anchors", like in GitHub graphs (ex: https://github.com/jhipster/jhipster-lite/actions/runs/3254882914. Those anchors are really good for readability.

They should react like the related connectors

### PR DESCRIPTION
**Pull request by abdelfetah18**
Fix #3966

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
can you resolve current conflict in this branch plz @abdelfetah18 ?

@Gnuk : any chance you have a quick look on this plz?

File: src/main/style/atom/chip/_chip.scss [hunk 1]
```diff
@@ -55,6 +56,51 @@
   &.-extended {
     margin-bottom: 15px;
   }
+  
+  &.-left-anchor-point::before {
```
Comment (Gnuk) [inline src/main/style/atom/chip/_chip.scss:60]: A lot of code seems to be duplicated in the next sections, maybe it's possible to mutualize. Also, lot of magic values with same value, maybe it's possible to add variables to be more explicit.

**Review by Gnuk — CHANGES_REQUESTED** [review]

File: src/main/style/atom/chip/_chip.scss (hunk 1 above)
Comment (abdelfetah18) [reply src/main/style/atom/chip/_chip.scss:60]: because of me using psuedo element when i try to avoid it,
the psuedo elements stops working

Comment (abdelfetah18) [reply src/main/style/atom/chip/_chip.scss:60]: and if i make it like this:
```sass
&.-anchor-point {
    position: absolute;
    top: calc(50% - 6px);
    outline: solid 1px var(--jhlite-chip-default-anchor-color);
    border: 3px solid var(--jhlite-chip-bg-color);
    border-radius: 50%;
    background-color: var(--jhlite-chip-default-anchor-color);
    width: 6px;
    height: 6px;
    content: '';
    
    &--left::before {
      left: -6px;
    }
  
    &--right::after {
      right: -6px;
    }
  }
```

the generated css code will look like this:
```sass
.jhlite-chip.-anchor-point {
  position: absolute;
  top: calc(50% - 6px);
  outline: solid 1px var(--jhlite-chip-default-anchor-color);
  border: 3px solid var(--jhlite-chip-bg-color);
  border-radius: 50%;
  background-color: var(--jhlite-chip-default-anchor-color);
  width: 6px;
  height: 6px;
  content: "";
}
.jhlite-chip.-anchor-point--left::before {
  left: -6px;
}
.jhlite-chip.-anchor-point--right::after {
  right: -6px;
}
```
which makes the styles on psuedo elements not working

Comment (abdelfetah18) [reply src/main/style/atom/chip/_chip.scss:60]: I found a solution using `@extend`, which is:

Putting this on the top of the file:
```sass
%anchor-point {
  position: absolute;
  top: calc(50% - 6px);
  outline: solid 1px var(--jhlite-chip-default-anchor-color);
  border: 3px solid var(--jhlite-chip-bg-color);
  border-radius: 50%;
  background-color: var(--jhlite-chip-default-anchor-color);
  width: 6px;
  height: 6px;
  content: '';
}
```

and then 
```sass
&.-left-anchor-point::before {
  @extend %anchor-point;
  left: -6px;
}
  
&.-right-anchor-point::after {
  @extend %anchor-point;
  right: -6px;
}
```

@Gnuk, Is this good ?

Comment (Gnuk) [reply src/main/style/atom/chip/_chip.scss:75]: Yes, it's better

**Comment by pascalgrimaud** [comment]
Thanks @abdelfetah18 for your work and your patience

**Comment by abdelfetah18** [comment]
> Thanks @abdelfetah18 for your work and your patience

You're very welcome

**Pull request merged**